public/styles/components/_footer.scss

```diff
@@ -0,0 +1,236 @@
+      margin-top: 0;
+
+      + .bsds-nav {
+        margin-top: space(lg);
+      }
+    }
+  }
+
+  .bsds-nav {
+    margin: space(6x) 0 0 0;
+    padding: 0;
+  }
+
+  .bsds-nav-item {
+    list-style: none;
+  }
+
+  .bsds-link-nav {
+    @include link-navigation;
+  }
+
+  .bsds-footer-base {
+    padding-bottom: space(6x);
+
+    .bsds-footer-base-container {
+      @include font-size($fs-body-subtle);
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: flex-start;
+      padding: 0 space(3x);
+
+      .bsds-footer-logo {
+        width: auto;
+        max-height: space(8x);
+        margin-top: space(3x);
+      }
+
+      .bsds-footer-text-small {
+        @include bsds-footer-text;
+      }
+
+      .bsds-footer-link-corp {
+        @include link-navigation;
+        @include bsds-footer-text;
+
+        .bsds-footer-icon {
+          width: space(lg);
+        }
+      }
+
+      .bsds-footer-link-cookie {
+        margin-top: space(lg);
+      }
+    }
+
+    .bsds-nav-footer-legal {
+      width: 100%;
+      border-bottom: $border-width-xxs solid $border-color-subtle;
+      padding: space(3x) 0;
+
+      .bsds-nav {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: flex-start;
+        margin: 0;
+        padding-left: space(3x);
+        list-style: none;
+
+        .bsds-nav-item {
+          .bsds-nav-link {
+            @include link-navigation;
+            @include font-size($fs-small);
+          }
+        }
+      }
+    }
+
+    .bsds-footer-social-media {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      padding-left: 0;
+      list-style: none;
+      gap: space(lg);
+
+      .bsds-footer-social-media-item {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin: 0;
+
+        .bsds-footer-social-media-icon {
+          max-height: space(2x);
+        }
+      }
+    }
+  }
+}
+
+@media (min-width: $breakpoint-md-min) {
+  .bsds-nav-footer {
+    display: flex;
+    padding: space(6x) space(3x);
+
+    > .bsds-nav-footer-column {
+      flex: 1;
+    }
+
+    .bsds-nav {
+      &:first-of-type {
+        margin-top: 0;
+      }
+    }
+
+    .bsds-nav-footer-column {
+      > .bsds-nav {
+        padding-right: 0;
+        padding-left: 0;
+      }
+
+      &:first-of-type {
+        > .bsds-nav {
+          &:first-of-type {
+            padding-top: 0;
+          }
+        }
+      }
+
+      &:last-of-type {
+        > .bsds-nav {
+          &:last-of-type {
+            padding-bottom: 0;
+          }
+        }
+      }
+    }
+  }
+
+  .bsds-nav-group {
+    &:not(:first-of-type) {
+      margin-top: space(6x);
+    }
+  }
+}
+
+@media (min-width: $breakpoint-lg-min) {
+  .bsds-footer {
+    .bsds-nav-footer {
+      border-bottom: 0;
+    }
+
+    .bsds-footer-base {
+      margin-top: space(4x);
```

Comment:
thoughts on moving this up to bsds-footer since if someone adds a class on the component overall, that's where it'd get applied?

let's also validate spacing between/around the two sections and the component overall. looks like there's extra between nav-footer and footer-base